src/index.d.ts

```diff
@@ -492,10 +492,16 @@ declare namespace admin.auth {
 
     /**
      * The date the user was created, formatted as a UTC string.
-     *
      */
     creationTime: string;
 
+    /**
+     * The time at which the user was last active (ID token refreshed), or null
```

Comment:
"The time at which the user was last active (ID token refreshed), formatted as a UTC Date string (e.g., 'Sat, 03 Feb 2001 04:05:06 GMT'). Returns null if the user was never active."

Reply:
Done.